docs/home/api-info/webhooks.mdx

```diff
@@ -0,0 +1,130 @@
+obtain such information.
+
+Webhooks turn this around: the producer of information calls the consumer when data is available.
+This way, the consumer doesn't have to poll the producer regularly to obtain information.
+
+Metriport will call your app's webhook endpoint when user data is available - there's no way for your
+application to request data exposed through webhooks.
+
+To enable this integration approach with Metriport:
+
+1. Expose a public endpoint on your app;
+1. Set this endpoint URL on the [Developers page](https://dash.metriport.com/developers) in the developer dashboard,
+   or via the [Update Settings endpoint](/home/api-reference/settings/post-settings);
+1. This will generate a webhook key that should be used to authenticate requests on
+   your app's endpoint (webhook).
+
+General requirements for the endpoint:
+
+1. Public endpoint accessible from the internet;
+1. Accepts a `POST` HTTP request;
+1. Verifies requests by comparing the HTTP header `x-webhook-key` with the webhook key;
+1. Accepts and responds to a [`ping` message](#the-ping-message);
+
+Additionally, depending on what Metriport APIs you're using, your endpoint will need to accept and process
+different messages when they become available - specifically:
+
+1. If using the Devices API - accept and process [user data messages](#devices-api-messages).
+1. If using the Medical API - accept and process [patient data messages](#medical-api-messages).
+
+## Authentication
+
+Validate requests to your public endpoint by comparing the webhook key with each request's HTTP header.
+
+The webhook key is accessible on [Developers page](https://dash.metriport.com/developers) in the developer
+dashboard, or via the [Get Settings endpoint](/home/api-reference/settings/get-settings) - see how to enable webhooks on the [overview section](#overview).
+
+The HTTP header will contain a header named `x-webhook-key` - it should contain the webhook key mentioned above.
+Your app is expected to disregard requests if those don't match. We suggest returning a HTTP status code
+`401` with no body in such cases.
+
+### Generating a new webhook key
+
+If using the dashboard: simply delete your webhook URL on the [Developers page](https://dash.metriport.com/developers), save, and enter it again.
+
+If using the API: set the webhook URL to an empty string via the [Update Settings endpoint](/home/api-reference/settings/post-settings), and then set it
+to your desired URL making another request to the same endpoint.
+
+## Format
+
+Webhook requests contain the relevant information on the body of the HTTP request.
+
+There are three types of messages you can expect to receive:
+
+- [`ping`](#the-ping-message): validation of the webhook connection between Metriport and your app;
+- [user data](#devices-api): health information grouped by user (Devices API only).
+- [patient data](#medical-api): patient document information (Medical API only).
```

Comment:
Wouldn't be better to point the user to this page's section of the respective API, like we do at the end of the `Overview` section?